samples/dotnet/kernel-syntax-examples/Example26_SemanticFunctionsUsingChatGPT.cs

```diff
@@ -0,0 +1,36 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System;
+using System.Threading.Tasks;
+using Microsoft.SemanticKernel;
+using RepoUtils;
+
+/**
+ * This example shows how to use GPT3.5 Chat model for prompts and semantic functions.
```

Comment:
GPT-4 + all future models are currently planned to only be chat-completions too aren't they?

Reply:
I wouldn't take that for granted. Currently there's a lot of attention for chat scenarios, and I wouldn't be surprised if this shifts back to text once explorations move from UI to scalability, e.g. complex semantic memory